File: src/string/convertToPythonCase/convert_to_python_case.py
def convert_to_python_case(input):
    snake_list = []
    for i in range(len(input)):
        if input[i].isupper():
            if i > 0:
                snake_list.append('_')
            snake_list.append(input[i].lower())
        else:
            snake_list.append(input[i])
    result = ''.join(snake_list)
    print(result) 

File: src/string/convertToPythonCase/ConvertToPythonCase.java
package string.convertToPythonCase;

import java.util.stream.Collectors;

public class ConvertToPythonCase {
    public static void convertToPythonCase(String input) {
        String result = input.chars()
                .mapToObj(i -> (char) i)
                .map(ch -> Character.isUpperCase(ch)
                        ? "_" + String.valueOf(ch).toLowerCase()
                        : String.valueOf(ch))
                .collect(Collectors.joining());

        if (result.startsWith("_")) {
            result = result.substring(1);
        }
        System.out.println(result);
    }

}